recipes/espsim/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "espsim" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # NOTE(hadim): waiting for https://github.com/hesther/espsim/issues/15#issuecomment-1685256208
+  url: https://github.com/hesther/espsim/archive/533a53105858d1a25cbe2fb54dd7208d7989caa1.zip
+  sha256: 55f2026f39db7cec1fd091c41233af1343d732b97a667a28a03810099143a51f
```

Comment:
Waiting for https://github.com/hesther/espsim/issues/15#issuecomment-1685256208 for a proper git tag to use.